content/influxdb/cloud-dedicated/reference/sql/operators/logical.md

```diff
@@ -0,0 +1,463 @@
+| --: | ---: | :---------- | ---: | :------------------- |
+|   0 | 35.9 | Living Room | 21.1 | 2022-01-01T08:00:00Z |
+|   0 | 35.9 | Kitchen     |   21 | 2022-01-01T08:00:00Z |
+|   0 | 35.9 | Living Room | 21.4 | 2022-01-01T09:00:00Z |
+|   0 | 36.2 | Kitchen     |   23 | 2022-01-01T09:00:00Z |
+
+{{% /influxdb/custom-timestamps %}}
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+## LIKE {.monospace}
+
+The `LIKE` operator returns `true` if the left operand matches the string pattern
+specified in the right operand.
+`LIKE` expressions support [SQL wildcard characters](#sql-wildcard-characters).
+
+{{< flex >}}
+{{% flex-content "two-thirds operator-example" %}}
+
+```sql
+SELECT 'John' LIKE 'J_%n' AS "LIKE condition"
+```
+
+{{% /flex-content %}}
+{{% flex-content "third operator-example" %}}
+
+| LIKE condition |
+| :------------- |
+| true           |
+
+{{% /flex-content %}}
+{{< /flex >}}
+
+{{< expand-wrapper >}}
+{{% expand "`LIKE` operator in the `WHERE` clause" %}}
+
+```sql
+SELECT *
+FROM home
+WHERE
+  room LIKE '%Room'
+LIMIT 4
+```
+
+{{% influxdb/custom-timestamps %}}
+
+|  co |  hum | room        | temp | time                 |
+| --: | ---: | :---------- | ---: | :------------------- |
+|   0 | 35.9 | Living Room | 21.1 | 2022-01-01T08:00:00Z |
+|   0 | 35.9 | Living Room | 21.4 | 2022-01-01T09:00:00Z |
+|   0 |   36 | Living Room | 21.8 | 2022-01-01T10:00:00Z |
+|   0 |   36 | Living Room | 22.2 | 2022-01-01T11:00:00Z |
+
+{{% /influxdb/custom-timestamps %}}
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+### SQL wildcard characters
+
+The InfluxDB SQL implementation supports the following wildcard characters when
+using the `LIKE` operator to match strings to a pattern.
+
+| Character | Description                        |
+| :-------: | :--------------------------------- |
+|    `%`    | Represents zero or more characters |
+|    `_`    | Represents any single character    |
+
+## NOT {.monospace}
+
+The `NOT` operator negates the subsequent expression.
+
+{{< flex >}}
+{{% flex-content "two-thirds operator-example" %}}
+
+```sql
+SELECT NOT true AS "NOT condition"
+```
+
+{{% /flex-content %}}
+{{% flex-content "third operator-example" %}}
+
+| NOT condition |
+| :------------ |
+| false         |
+
+{{% /flex-content %}}
+{{< /flex >}}
+
+##### Examples
+
+{{< expand-wrapper >}}
+{{% expand "`NOT IN`" %}}
+
+```sql
+SELECT *
+FROM home
+WHERE
+  room NOT IN ('Kitchen', 'Bathroom')
+LIMIT 4
+```
+
+{{% influxdb/custom-timestamps %}}
+
+|  co |  hum | room        | temp | time                 |
+| --: | ---: | :---------- | ---: | :------------------- |
+|   0 | 35.9 | Living Room | 21.1 | 2022-01-01T08:00:00Z |
+|   0 | 35.9 | Living Room | 21.4 | 2022-01-01T09:00:00Z |
+|   0 |   36 | Living Room | 21.8 | 2022-01-01T10:00:00Z |
+|   0 |   36 | Living Room | 22.2 | 2022-01-01T11:00:00Z |
+
+{{% /influxdb/custom-timestamps %}}
+{{% /expand %}}
+
+{{% expand "`NOT EXISTS`" %}}
+
+```sql
+SELECT *
+FROM
+  home home_actions
```

Comment:
This doesn't require a comma?

Reply:
No, it doesn't. This is the syntax for a correlated subquery: https://arrow.apache.org/datafusion/user-guide/sql/subqueries.html#exists